apps/condo/domains/user/components/CookieAgreement.tsx

```diff
@@ -0,0 +1,94 @@
+import styled from '@emotion/styled'
+import cookie from 'js-cookie'
+import React, { useState } from 'react'
+
+import { ChevronDown } from '@open-condo/icons'
+import { useIntl } from '@open-condo/next/intl'
+import { Typography, Button, Space } from '@open-condo/ui'
+
+import { useContainerSize } from '@condo/domains/common/hooks/useContainerSize'
+
+const COOKIE_AGREEMENT_KEY = 'cookieAgreementAccepted'
+
+const StyledFixedPanel = styled.div`
+  position: fixed;
+  bottom: 0;
+  left: 0;
+  width: 100%;
+  box-shadow: 0 14px 34px rgba(208, 216, 225, 0.4), 0 15px 38px rgba(208, 216, 225, 0.4), 0 7px 17px rgba(208, 216, 225, 0.6);
+  z-index: 1;
+  background: white;
+  display: flex;
+  justify-content: center;
+`
+
+const StyledFixedPanelContent = styled(Space)`
+  justify-content: ${({ direction }) => direction === 'horizontal' ? 'space-between' : 'flex-start'};
+  padding: ${({ direction }) => direction === 'horizontal' ? '12px 40px' : '16px 20px'};
+  width: ${({ direction }) => direction === 'horizontal' ? '100%' : '640px'};
+  align-items: center;
+  box-sizing: border-box;
+`
+
+const toggleMoreTextStyle: React.CSSProperties = {
+    flexShrink: 0,
+    display: 'flex',
+    alignItems: 'center',
+}
+
+// When width will be less than this value, layout of auth-pages turns to vertical mode
+const MIN_CONTENT_WIDTH_FOR_DESKTOP_MODE = 990
+
+export const CookieAgreement = () => {
+    const intl = useIntl()
+    const CookieAgreementTitleMsg = intl.formatMessage({ id: 'component.CookieAgreement.title' })
+    const CookieAgreementDescriptionMsg = intl.formatMessage({ id: 'component.CookieAgreement.description' })
+    const CookieExpandActionMsg = intl.formatMessage({ id: 'component.CookieAgreement.action.expand' })
+    const CookieAcceptActionMsg = intl.formatMessage({ id: 'component.CookieAgreement.action.accept' })
+    const [displayMore, setDisplayMore] = useState(false)
```

Comment:
We should try to separate component code by special groups:
1) useIntl and Internalisation constants
2) hooks
3) local component state
4) local component handlers such as click or change
5) render constants (like a global loading state and etc)